frontend/src/app/projects/project-overview/project-overview.component.ts

```diff
@@ -33,7 +33,7 @@ export class ProjectOverviewComponent implements OnInit {
       .list()
       .pipe(
         tap(() => {
-          this.showSpinner = false;
+          this.loading = false;
         })
```

Comment:
Here the curly brackets can be removed and the `next` and `error` case of the subscription are the same (which should not be the case IMO

Reply:
Removed the curly brackets. `next` and `error` are not the same on my side.

Reply:
Yes that was my mistake somehow I have seen no difference